sw/airborne/subsystems/radio_control.c

```diff
@@ -64,7 +64,11 @@ void radio_control_periodic_task(void)
     radio_control.time_since_last_frame++;
   }
 
-#if defined RADIO_CONTROL_LED
+#ifndef RADIO_CONTROL_LED
```

Comment:
What is this about?
`RADIO_CONTROL_LED` should simply be not defined if you don't have one...